rockstor.spec

```diff
@@ -447,6 +447,20 @@ exit 0
 # https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 # https://github.com/openSUSE/obs-build/blob/master/changelog2spec
 %changelog
+* Tue Nov 19 2024 The Rockstor Project <support@rockstor.com> - 5.0.15-0
+-@cellisten made their first contribution 

```

Comment:
```suggestion
```

Reply:
Removing autogenerated changelog entry as we use this within our Web-UI changelog and may end up breaking at least formatting there as it does not conform to the usual format.